src/Mumble.proto

```diff
@@ -184,9 +184,14 @@ message UserState {
 	optional bool self_deaf = 10;
 	// User image if it is less than 128 bytes.
 	optional bytes texture = 11;
-	// TODO ??
+	// The plugin identifier, currently used for positional audio.
+	// Positional audio information is only sent to users who share
+	// identical plugin contexts.
+	//
+	// This value is not trasmitted to clients.
 	optional bytes plugin_context = 12;
-	// TODO ??
+	// The user's plugin-specific identity (currently unused).
```

Comment:
This field is used to transmit identity information to the server. See Plugins.cpp:443 . It is never transmitted back to clients though. That much is correct. If you meant that the server isn't processing it itself that's true too but misleading as this information is used and meant to be used by Ice scripts. E.g. https://github.com/mumble-voip/mumo uses it for the bf2 and source functionality. There the identity field contains json with lots of information like team, squad leadership and so on.